### PR TITLE
Page spreads implemented

### LINKED ISSUE #900: Odd/Even Spreads option for Page Mode
https://forums.zotero.org/discussion/107448/ios-page-spreads

Are we able to do this?

### PR DESCRIPTION
**Pull request by michalrentka**
Closes #900

### CODE REVIEW

File: Zotero/Scenes/Detail/PDF/Models/PDFSettings.swift [hunk 1]
```diff
@@ -35,12 +44,14 @@ extension PDFSettings: Codable {
         let appearanceRaw = try container.decode(UInt.self, forKey: .appearanceMode)
         let modeRaw = (try? container.decode(UInt.self, forKey: .pageMode)) ?? 2
         let fittingRaw = (try? container.decode(Int.self, forKey: .pageFitting)) ?? 2
+        let isFirstPageAlwaysSingle = (try? container.decode(Bool.self, forKey: .isFirstPageAlwaysSingle)) ?? false
```
Comment (mvasilak) [inline Zotero/Scenes/Detail/PDF/Models/PDFSettings.swift:47]: We should use the PSPDFKIt default here, which is `true`.

**Review by mvasilak — COMMENTED** [review]
One minor change. 
I undestand that `Even` (`true`) is `Even-Odd` and `Odd` (`false`) is `Odd-Even`.
Could the full names be more clear for users?

**Comment by michalrentka** [comment]
> One minor change. I undestand that `Even` (`true`) is `Even-Odd` and `Odd` (`false`) is `Odd-Even`. Could the full names be more clear for users?

@dstillman what do you think?

File: Zotero/Scenes/Detail/PDF/Models/PDFSettings.swift (hunk 1 above)
Comment (michalrentka) [reply Zotero/Scenes/Detail/PDF/Models/PDFSettings.swift:47]: You're right, I got that mixed up in here, got it right just few lines above 😅

**Pull request merged**